### PR TITLE
mitx/dogwood.3-1rc1

### PR DESCRIPTION
**Pull request by blarghmatey**
Cherry picked commits to enable various features and fix a selection of bugs.

Addresses mitodl/salt-ops#72 mitodl/salt-ops#82 and mitodl/salt-ops#83

### CODE REVIEW

File: requirements/edx/edx-private.txt [hunk 1]
```diff
@@ -26,7 +26,7 @@
 -e git+https://github.com/pmitros/ProfileXBlock.git@4aeaa24aa2bc7d9cb2d2bb60d6f05def3b856be0#egg=profile-xblock
 
 # Peer instruction xblock - prototype (from UBC; not for use on edx.org yet)
```
Comment (pdpinch) [inline requirements/edx/edx-private.txt]: I just realized that this change is to edx-private requirements. Do we actually use those, or do we need to move this elsewhere?

**Comment by pdpinch** [comment]
Could you add an links to the relevant issues in the description? I know this addresses MathJax, the orphan export problem, the LTI XBlock and the Peer Instruction XBlock. Is that it?

File: requirements/edx/edx-private.txt (hunk 1 above)
Comment (blarghmatey) [inline requirements/edx/edx-private.txt]: I believe that those do get used in our deployment.

**Comment by blarghmatey** [comment]
Can someone give me a +1 on this? @itsbenweeks @pdpinch @bdero

File: scripts/create-dev-env.sh [hunk 2]
```diff
@@ -422,7 +422,7 @@ fi
 # compile numpy and scipy if requested
```
Comment (itsbenweeks) [inline scripts/create-dev-env.sh]: Was this the line responsible for your scipy headaches? It looks like the other requirements files above are installing anything above 0.18.

**Comment by itsbenweeks** [comment]
Aside from the one scipy compilation oddity, this looks good to me @blarghmatey.

File: scripts/create-dev-env.sh (hunk 2 above)
Comment (blarghmatey) [inline scripts/create-dev-env.sh]: No, this script never actually gets run and apparently isn't used by anyone anymore. Ned opened a PR on edx/edx-platform to remove it entirely.

**Comment by pdpinch** [comment]
This is what I was looking for: https://github.com/mitodl/edx-platform/blob/25aa65658e626229e9b98acd50a371aabc6a51a7/requirements/edx/github.txt#L98

`git+https://github.com/edx/xblock-lti-consumer.git@v1.0.0#egg=xblock-lti-consumer==v1.0.0`

👍

**Pull request merged**